profiles/common/modules/features/nexteuropa_communities/nexteuropa_communities.info

```diff
@@ -2,22 +2,36 @@ name = Nexteuropa Communities
 description = Manage communities in Nexteuropa
 core = 7.x
 package = NextEuropa
+dependencies[] = cce_basic_config
 dependencies[] = context_og
+dependencies[] = ctools
 dependencies[] = entityreference_prepopulate
+dependencies[] = features
+dependencies[] = field_group
+dependencies[] = list
+dependencies[] = menu
```

Comment:
I think this rule is now outdated... I think it was to avoid to automatically include an existing menu (main-menu...) or menu item, but now, with the QA process, if an existing element is include, you detect it